order_events/email_service/service.bal

```diff
@@ -0,0 +1,24 @@
+import ballerinax/kafka;
+import ballerina/io;
+import wso2/orders.types;
+
+listener kafka:Listener orderListener = new (kafka:DEFAULT_URL, {
+    groupId: "email_group",
+    topics: "inventory_updated_topic"
+});
+
+service on orderListener {
+    private final kafka:Producer orderProducer;
```

Comment:
When declaring client inside a service, shall we follow a consistent approach ? (check for other places you have declared db-client, slack-client etc.)